### ISSUE TITLE
Google Maps widget not interactive

### ISSUE THREAD

**Issue by tomas-eklund** [elementor/elementor#8244]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The Google Maps widget is no longer interactive - you can't zoom, pan or otherwise interact with the map as you usually could. We have noticed this ourselves on several sites and some of our clients have complained. 

If you insert the same map manually using the HTML widget, it works.

The problem seems to lie with the generated CSS rule applied to the element:
```
.elementor-#postid# .elementor-element.elementor-element-#elementid# iframe { 
	pointer-events: none 
}
```

## Steps to reproduce

1. Insert the Elementor Google Maps widget on a page and save. 
2. Navigate to the page. 
3. Try to interact with the map. It does not work.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

http://www.elementor-demo.eu/bug-reports/google-maps/ 

## Environment

<details>
<summary>System Info</summary>
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.6.43-84.3
	PHP Version: 7.2.18
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.2.1
	Site URL: http://www.elementor-demo.eu
	Home URL: http://www.elementor-demo.eu
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Stockholm
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.0.6
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.5.16
		Author: Elementor.com

	Elementor Pro
		Version: 2.5.9
		Author: Elementor.com

== Log ==
	: 
Log: showing 20 of 312019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_posts Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_portfolio Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_products Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_products Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_sitemap Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_0_popup_border_radius Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_posts Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_posts Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_portfolio Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_portfolio Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_products Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_products Finished 
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_form Start  
2019-05-21 10:23:56 [info] Elementor Pro/Upgrades - _v_2_5_4_form Finished 
2019-05-21 10:23:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.2',
  'to' => '2.5.8',
)]
2019-05-31 17:01:47 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.15',
  'to' => '2.5.16',
)]
2019-05-31 17:02:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.5.8',
  'to' => '2.5.9',
)]

PHP: showing 1 of 1PHP: 2019-02-25 14:11:44 [notice X 1][/home/vhosts/elementor-demo.eu/httpdocs/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::95] Undefined index: form_fields [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

</details>

**Comment by tomas-eklund** [comment]
### Workaround 

Apply the following custom CSS to the Maps Widget

```
selector iframe {
    pointer-events: auto;
}
```

**Comment by bainternet** [comment]
@tomas-eklund 	

Good news: In 2.6.0 it's already removed.

A little background, we added it for a reason which caused an error on page scrolling, it would zoom instead of scrolling, Google Maps have changed that and so did we, like I said in the next version. as for zooming, you can already do that by setting "prevent scroll" to no.

thanks.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v2.6.0

Feel free to update

Thanks!
